deployments/container/native-only.mk

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 PUSH_ON_BUILD ?= false
-DOCKER_BUILD_PLATFORM_OPTIONS = --platform=linux/amd64
+DOCKER_BUILD_PLATFORM_OPTIONS = --platform=linux/aarch64
```

Comment:
I don't think this change should go in like this. If we change it at all we should change it to detect the platform you are on, not just change it to a different (hard-coded) one.

Reply:
sorry, this was not meant to be part of the changes

Reply:
Fixed